travis/before-install-maven

```diff
@@ -1,8 +1,6 @@
 #!/bin/sh -eux
 
-./gradlew prepareMavenBuild
-
-git clone --depth=1 https://github.com/secure-software-engineering/DroidBench.git /tmp/DroidBench
+./gradlew prepareMavenBuild install -x javadoc
```

Comment:
Ah, nice idea to exclude `javadoc` here!

Reply:
Yes, I also exclude javadoc when running the main Maven build.  We generate javadoc for everything during the Gradle build, so no need to do it here.  And it's surprising how expensive javadoc generation is!  It takes longer than compilation.